xonsh/environ.py

```diff
@@ -1197,6 +1198,17 @@ def tracer(cmds: list, captured: Union[bool, str]):
         "or None / the empty string if traceback logging is not desired. "
         "Logging to a file is not enabled by default.",
     )
+    XONTRIBS_AUTOLOAD_DISABLED = Var(
```

Comment:
I think it's probably clearer to name this `XONTRIBS_AUTOLOAD` or `XONTRIBS_AUTOLOAD_ENABLED` and default it to `True`

Reply:
Yes I thought about that naming, when setting this to a list of names, it has to make sense.

Reply:
Ahh, that's a good point.  Ok, all set!